locale/fi/LC_MESSAGES/django.po

```diff
@@ -3703,3 +3698,6 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:78
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+#~ msgid "Password"
```

Comment:
Can you run `makemessages` with option `--no-obsolete` thanks. Sorry, this is missing from the guides..

Reply:
Would that mean that the translation is lost if in the future one would add to add something with the text "Password"?

Reply:
Yes, but we have git so they could find the translation from the history.